applications/nrf_desktop/src/hw_interface/board.c

```diff
@@ -21,24 +21,23 @@
 LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BOARD_LOG_LEVEL);
 
 
-static int port_setup(const char *name,
+static int port_setup(const struct device *port,
 		      const struct pin_state pin_state[],
 		      size_t cnt)
 {
-	const struct device *gpio_dev = device_get_binding(name);
 	int err = 0;
 
-	if (!gpio_dev) {
+	if (!device_is_ready(port)) {
 		LOG_ERR("Cannot bind %s", name);
```

Comment:
This won't compile. Please update it too.

Reply:
oops, fixed